servers/src/main/java/tachyon/worker/block/meta/StorageDir.java

```diff
@@ -373,7 +373,7 @@ public void removeTempBlockMeta(TempBlockMeta tempBlockMeta) throws NotFoundExce
   }
 
   /**
-   * Changes the size of a temp block or throws IOException.
+   * Changes the size of a temp block or throws Exception.
    *
```

Comment:
Well, I do agree with you! I didn't think of that and just found that the original comment is unsuitable. I will remove it. thanks for your comment! @uronce-cc